Tests/SotoCoreTests/AWSClientTests.swift

```diff
@@ -261,217 +261,217 @@ class AWSClientTests: XCTestCase {
-                XCTAssertEqual(request.headers["Content-Length"], bufferSize.description)
-                let requestData = request.body.getData(at: 0, length: request.body.readableBytes)
-                XCTAssertEqual(requestData, data)
-                let response = AWSTestServer.Response(httpStatus: .ok, headers: [:], body: nil)
-                return .result(response)
-            }
-
-            try await responseTask
-        } catch let error as AWSClient.ClientError where error == .tooMuchData {
-        } catch {
-            XCTFail("Unexpected error: \(error)")
-        }
-    }
-
-    func testRequestChunkedStreaming() async throws {
-        struct Input: AWSEncodableShape & AWSShapeWithPayload {
-            static var _payloadPath: String = "payload"
-            static var _options: AWSShapeOptions = [.allowStreaming, .allowChunkedStreaming, .rawPayload]
-            let payload: AWSPayload
-            private enum CodingKeys: CodingKey {}
-        }
-
-        let awsServer = AWSTestServer(serviceProtocol: .json)
-        let httpClient = HTTPClient(eventLoopGroupProvider: .createNew)
-        let config = createServiceConfig(endpoint: awsServer.address)
-        let client = createAWSClient(credentialProvider: .empty, httpClientProvider: .shared(httpClient))
-        defer {
-            XCTAssertNoThrow(try awsServer.stop())
-            XCTAssertNoThrow(try client.syncShutdown())
-            XCTAssertNoThrow(try httpClient.syncShutdown())
-        }
-        do {
-            // supply buffer in 16k blocks
-            let bufferSize = 145 * 1024
-            let blockSize = 16 * 1024
-            let data = createRandomBuffer(45, 9182, size: bufferSize)
-            var byteBuffer = ByteBufferAllocator().buffer(capacity: bufferSize)
-            byteBuffer.writeBytes(data)
-
-            let payload = AWSPayload.stream { eventLoop in
-                let size = min(blockSize, byteBuffer.readableBytes)
-                if size == 0 {
-                    return eventLoop.makeSucceededFuture(.end)
-                } else {
-                    return eventLoop.makeSucceededFuture(.byteBuffer(byteBuffer.readSlice(length: size)!))
-                }
-            }
-            let input = Input(payload: payload)
-            async let responseTask: Void = client.execute(operation: "test", path: "/", httpMethod: .POST, serviceConfig: config, input: input, logger: TestEnvironment.logger)
-
-            try awsServer.processRaw { request in
-                let bytes = request.body.getBytes(at: 0, length: request.body.readableBytes)
-                XCTAssertTrue(bytes == data)
-                let response = AWSTestServer.Response(httpStatus: .ok, headers: [:], body: nil)
-                return .result(response)
-            }
-
-            try await responseTask
-        } catch {
-            XCTFail("Unexpected error: \(error)")
-        }
-    }
+    /*    func testRequestStreaming() async throws {
```

Comment:
Is this to be added back in or deleted?

Reply:
I should have made it clear streaming requests/responses don't work yet. They will the subject of the next few PRs. So I commented out the tests that would fail and will uncomment them as features are implemented.